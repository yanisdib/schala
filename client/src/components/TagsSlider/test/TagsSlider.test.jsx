import { shallow } from 'enzyme';

import { TagsSlider } from '../..';


describe('TagsSlider component', () => {
    let wrapper;

    beforeEach(() => wrapper = shallow(<TagsSlider></TagsSlider>))

    it('renders correctly', () => expect(wrapper).toMatchSnapshot())
})