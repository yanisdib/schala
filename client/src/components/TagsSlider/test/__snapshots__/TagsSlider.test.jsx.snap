// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagsSlider component renders correctly 1`] = `
<div
  className="tags-wrapper"
>
  <styled.div
    translateValue={0}
  >
    <Tags />
  </styled.div>
  <SliderNavigationButton
    isNext={true}
    onClick={[Function]}
    size={25}
  />
</div>
`;
